### PR TITLE
Store return forms as separate metadata

### PR DESCRIPTION
**Pull request by longhotsummer**

### CODE REVIEW

**Comment by longhotsummer** [comment]
@jbothma when you get a chance, please sanity check this.

File: models/aged_creditor.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "aged_creditor_facts",
     "label": "Aged Creditor Analysis",
-    "description": "This is based on the AC and ACA return forms.",
+    "description": "Analysis of creditors by age of debt.",
```
Comment (jbothma) [inline models/aged_creditor.json]: Worth adding "This does not list individual creditors, but rather the amount of debt in each age group for each line item category"?

File: models/aged_debtor.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "aged_debtor_facts",
     "label": "Aged Debtor Analysis",
-    "description": "This is based on the AD and ADA return forms.",
+    "description": "Analysis of debtors by age of debt.",
```
Comment (jbothma) [inline models/aged_debtor.json]: Worth adding "This does not list individual creditors, but rather the amount of debt in each age group for each line item category"?

File: models/audit_opinions.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "fact_table": "audit_opinion_facts",
     "label": "Audit Opinions",
-    "description": "",
+    "description": "Opinions of financial performance from the Auditor General.",
```
Comment (jbothma) [inline models/audit_opinions.json]: isn't it opinions of audit result? I haven't read these but I didn't think they look at financial performance (in the sense we're looking at performance), but primarily "Are their finances recorded as they should be?"

File: models/conditional_grants.json [hunk 4]
```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "conditional_grants_facts",
     "label": "Conditional Grants",
-    "description": "This is based on the return forms for individual grants.",
+    "description": "Details of grants received from national government.",
+    "return_forms": ["Individual Grants"],
```
Comment (jbothma) [inline models/conditional_grants.json]: I don't think Individual Grants is a _term_. I was referring to each grant having an individual return form. See secrion _Conditional Grant Returns_ at https://mfmamirror.github.io/Return_Forms/

File: models/incexp.json [hunk 5]
```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "incexp_facts",
     "label": "Income and Expenditure",
-    "description": "Statement of financial performance. This is based on the OSB, OSR, OSA and OSAA return forms.",
+    "description": "Statement of Financial Position. How a municipality has spent money and received income.",
```
Comment (jbothma) [inline models/incexp.json]: _Financial Position_ is Balance Sheet.
_Financial Performance_ is Operating Income and Expenditure

File: models/repmaint.json [hunk 6]
```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "repmaint_facts",
     "label": "Repairs and Maintenance",
-    "description": "This is based on the RME return form.",
+    "description": "Money spent on repairs and maintenance of assets.",
+    "return_forms": ["RME"],
```
Comment (jbothma) [inline models/repmaint.json]: I think I was wrong in assuming this dataset is from these forms. I think Marina said last week that they pull this from another Schedule A form (not a Return Form)

**Comment by jbothma** [comment]
I like it but I think we have to fix some things

File: models/aged_creditor.json (hunk 1 above)
Comment (longhotsummer) [inline models/aged_creditor.json]: Nice idea.

File: models/audit_opinions.json (hunk 3 above)
Comment (longhotsummer) [inline models/audit_opinions.json]: It's like you're channeling Elsabe...

File: models/conditional_grants.json (hunk 4 above)
Comment (longhotsummer) [inline models/conditional_grants.json]: Yeah, I was trying to capture that into Individual Grants.

File: models/incexp.json (hunk 5 above)
Comment (longhotsummer) [inline models/incexp.json]: Fixed.

File: models/repmaint.json (hunk 6 above)
Comment (longhotsummer) [inline models/repmaint.json]: I'm beginning to think repmaint isn't useful, if we can get everything else from  capex.

Comment (jbothma) [inline models/repmaint.json]: I think I asked whether there's value in showing it in a dedicated cube. I think they said the value in this is the breakdown in this view while capex only shows the total

File: models/cflow.json [hunk 7]
```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "cflow_facts",
     "label": "Cash Flow",
-    "description": "This is based on the CFB, CFR, CFA and CFAA return forms.",
+    "description": "Statement of Financial Position. The movement of cash into and out of the municipality.",
```
Comment (jbothma) [inline models/cflow.json]: missed this earlier - I don't think this is Statement of Financial Position. I think only bsheet is that.

**Comment by jbothma** [comment]
Nice except for cflow description seeming incorrect

File: models/cflow.json (hunk 7 above)
Comment (longhotsummer) [inline models/cflow.json]: Updated.

**Comment by jbothma** [comment]
:+1:

**Pull request merged**